### PR TITLE
docs: content fixes pt 3

### PR DESCRIPTION
**Pull request by jillianhowarth**
Minor content fixes for data viz pages.

### CODE REVIEW

File: src/pages/data-visualization/advanced-charts/index.mdx [hunk 1]
```diff
@@ -131,16 +131,16 @@ neutral midpoint is often referred to as the "inflection point" (for example, a
 0 value midway between -100 and +100).
 
 [Divergent scales](../data-visualization/color-palettes#diverging-palettes)
-require 3 colors for their correspondent values (min, max and inflection point)
+require 3 colors for their correspondent values (min, max, and inflection point)
```
Comment (janchild) [inline src/pages/data-visualization/advanced-charts/index.mdx:134]:
```suggestion
require three colors for their correspondent values (min, max, and inflection point)
```

**Review by janchild — APPROVED** [review]
@jillianhowarth This is awesome! I just had one tiny tweak. 🎉

**Review by andreancardona — APPROVED** [review]
@jillianhowarth nice! 🎉

**Review by andreancardona — APPROVED** [review]
@jillianhowarth Looks good! 🎉

**Pull request merged**